sbncode/BeamSpillInfoRetriever/job/run_icarusbnbinfo_sbn.fcl

```diff
@@ -28,17 +28,17 @@ source: {
 
 physics: {
  producers: {
-   bnbinfo: @local::bnbspillinfo
+   icarusbnbinfo: @local::icarusbnbspillinfo
  }
 
- simulate: [bnbinfo ]
+ simulate: [icarusbnbinfo ]
  stream1: [ out1 ]
 }
 
 outputs: {
  out1: {
    module_type: RootOutput
-   fileName: "%ifb_%tc_bnbinfo.root"
+   fileName: "%ifb_%tc_icarusbnbinfo.root"
```

Comment:
Alternative proposal following the suggestion from SBNSoftware/icaruscode#823:
```suggestion
   fileName: "%ifb_%tc_bnbinfo.root"
```
(the change in this PR could actually affecting the workflow by changing the output file names)